Add changelog for PR #18484 (#22560)
ydb-platform/ydb: ydb/apps/ydb/CHANGELOG.md

> * Reworked paths approach in `ydb export s3` and `ydb import s3` commands.

Reviewer: ```suggestion
* Added a new paths approach in `ydb export s3` and `ydb import s3` commands with the new `--include` option instead of the `--item` option. 
```

> * Added export encryption features to `ydb export s3` and `ydb import s3` commands.

Reviewer: ```suggestion
* Added support for encryption features in the `ydb export s3` and `ydb import s3` commands.
```
